unified_planning/model/problem.py

```diff
@@ -471,6 +473,19 @@ def initial_values(self) -> Dict['up.model.fnode.FNode', 'up.model.fnode.FNode']
                     f_exp = self._get_ith_fluent_exp(f, domain_sizes, i)
                     res[f_exp] = self.initial_value(f_exp)
         return res
+    
+    def initial_defaults(self) -> Dict['up.model.types.Type', 'up.model.fnode.FNode']:
+        '''Returns the problem's initial defaults.'''
+        return self._initial_defaults
+    
+    def fluents_defaults(self) -> Dict['up.model.fluent.Fluent', 'up.model.fnode.FNode']:
+        '''Returns the problem's fluents defaults.'''
+        return self._fluents_defaults
+    
+    def _initial_values_structure(self) -> Dict['up.model.fnode.FNode', 'up.model.fnode.FNode']:
```

Comment:
This method is used outside the `Problem` class. We should remove the initial `_`.

I think also that `explicit_initial_values` could be a better name. What do you think?